docs/pipelines.md

```diff
@@ -330,7 +330,7 @@ The [`Parameters`](#specifying-parameters) are read from the `Pipeline` and [`Re
 
 The declared `WhenExpressions` are evaluated before the `Task` is run. If all the `WhenExpressions` evaluate to `True`, the `Task` is run. If any of the `WhenExpressions` evaluate to `False`, the `Task` is not run and the `Task` is listed in the [`Skipped Tasks` section of the `PipelineRunStatus`](pipelineruns.md#monitoring-execution-status). 
 
-In these examples, `create-readme-file` task will only be executed if the `path` parameter is `README.md` and `echo-file-exists` task will only be executed if the `status` result from `check-file` task is `exists`. 
+In these examples, `create-readme-file` task will only be executed if the `path` parameter is `README.md` and `echo-file-exists` task will only be executed if the `exists` result from `check-file` task is `yes`. 
```

Comment:
please replace `create-readme-file` with the pipeline task name `first-create-file`, its easier to follow through the pipeline task name instead of the task itself.

Reply:
makes sense, updated the task name